src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -213,53 +211,36 @@ const formConfig = {
       pages: {
         applicantNameDob: {
           path: 'applicant-information',
-          title: 'Applicant name and date of birth',
-          arrayPath: 'applicants',
+          title: formData =>
+            `${
+              formData.certifierRole === 'applicant' ? 'Your' : 'Applicant'
+            } name and date of birth`,
           uiSchema: applicantNameDobSchema.uiSchema,
           schema: applicantNameDobSchema.schema,
         },
-        applicantStart: {
-          path: 'applicant-information/:index/start',
-          arrayPath: 'applicants',
-          title: item => `${applicantWording(item)} information`,
-          showPagePerItem: true,
-          depends: () => !window.location.href.includes('review-and-submit'),
-          uiSchema: applicantStartSchema.uiSchema,
-          schema: applicantStartSchema.schema,
-        },
```

Comment:
This was an intro page that displayed before each applicant to introduce their swath of pages. No longer needed.